ietf-layer0-types-ext.yang

```diff
@@ -684,13 +724,46 @@ module ietf-layer0-types-ext {
          per second in a digitally 
          modulated signal or a line code";
     }
+
+    leaf nyquist-spacing-factor {
+      type decimal64 {
+        fraction-digits 4;
+        range "0..max";
+      }  
+      config false;
+      description 
+        "1.x factor to multiply the baud rate
+         for Nyquist shaped signal";

```

Comment:
Is this attribute encoding x or 1.x?

In the latter case, I think that the range should be "1..2" instead of "0..max"

Reply:
addressed with https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/22/files#r609845877